src/Groups/sub.jl

```diff
@@ -483,6 +483,10 @@ PermGroup
 """
 function quo(G::T, N::T) where T <: GAPGroup
   mp = GAP.Globals.NaturalHomomorphismByNormalSubgroup(G.X, N.X)::GapObj
+  # The call may have found out new information about `G.X`,
+  # for example that `G.X` is finite.
+#FIXME: The GAP function should deal with this situation.
```

Comment:
What GAP function should deal with what situation? You mean, if anything is learned about a group $G$ then all its "known" subgroups and quotients should be informed about it? This is something the Homalg packages implement for their objects, it is kind of like immediate methods on steroids (with all the pros and cons this implies).

Or maybe just that all "related groups" that are known "and at hand / reachable / already handled in our code" ? As in the case of `NaturalHomomorphismByNormalSubgroup`, but potentially others (e.g. `Normalizer`)?

Reply:
Sorry for not being precise enough.
I meant that `GAP.Globals.NaturalHomomorphismByNormalSubgroup` (or rather `GAP.Globals.NaturalHomomorphismByNormalSubgroupNC`) is a better place for the call of `UseSubsetRelation`, and I meant only one call for the pair `(G, N)`.